pytorch_lightning/trainer/training_loop.py

```diff
@@ -53,17 +52,17 @@ def __init__(
         self.running_loss = TensorRunningAccum(window_length=20)
         self._curr_step_result = None
         self._cur_grad_norm_dict = None
-        self._multiple_trainloader_mode = multiple_trainloader_mode
         self._skip_backward = False
-        self.trainer._multiple_trainloader_mode = multiple_trainloader_mode
         self._optimizer_freq_cumsum = None
+        self._hiddens = None
```

Comment:
typehints will also be helpful

Reply:
is it ok if we add typehints after major refactors?